config/locales/en.yml

```diff
@@ -104,3 +104,102 @@ en:
+      image_path: 'contributing/img-job.svg'
+      image_alt: Briefcase
+    - title: Impact
+      description: The features you work on or the lessons you create will help thousands of students learn life changing skills.
+      image_path: 'contributing/img-community.svg'
+      image_alt: Impact
+  hall_of_fame:
+    - name: Erik Trautman
+      image: hall_of_fame_images/erik.jpg
+      github_url: http://github.com/eriktrautman
+    - name: Joe Sawyer
+      image: hall_of_fame_images/joe.jpg
+      github_url: http://github.com/zkay
+    - name: Afshin Moktari
+      image: hall_of_fame_images/afshin.jpg
+      github_url: http://github.com/afshinator
+    - name: Nathan Hall
+      image: hall_of_fame_images/nathanhall.png
+      github_url: http://github.com/dominathan
+    - name: Abby Jones
+      image: hall_of_fame_images/abby.jpg
+      github_url: http://github.com/AbbyJonesDev
+    - name: Joshua Gorchov
+      image: hall_of_fame_images/joshua.jpg
+      github_url: http://github.com/gorchov
+    - name: Neil Gehani
+      image: hall_of_fame_images/neil.jpg
+      github_url: http://github.com/ngehani
+    - name: Ian White
+      image: hall_of_fame_images/ian.jpg
+      github_url: ''
+    - name: Bill Walker
+      image: hall_of_fame_images/bill.jpg
+      github_url: http://github.com/mach1010
+    - name: Mazin Power
+      image: https://avatars1.githubusercontent.com/u/7075351?v=3&s=400
+      github_url: https://github.com/muzfuz
+    - name: Ada Chiu
+      image: https://avatars0.githubusercontent.com/u/13413986?v=3&s=400
+      github_url: https://github.com/adachiu
+    - name: Kevin Mulhern
+      image: https://avatars1.githubusercontent.com/u/7963776?v=3&s=460
+      github_url: https://github.com/KevinMulhern
+    - name: Arun Kumar
+      image: https://avatars0.githubusercontent.com/u/7039523?v=3&s=400
+      github_url: https://github.com/arku
+    - name: Cody Loyd
+      image: https://avatars0.githubusercontent.com/u/18574792?v=3&s=400
+      github_url: https://github.com/codyloyd
+    - name: Cornelius Soon
+      image: https://avatars2.githubusercontent.com/u/16495574?v=3&s=400
+      github_url: https://github.com/csrail
+    - name: Rhys
+      image: https://avatars3.githubusercontent.com/u/19835799?v=3&s=400
+      github_url: https://github.com/105ron
+    - name: Austin
+      image: https://avatars2.githubusercontent.com/u/12649577?v=3&s=400
+      github_url: https://github.com/CouchofTomato
+    - name: Leo
+      image: https://avatars2.githubusercontent.com/u/13614862?v=3&s=400
+      github_url: https://github.com/leosoaivan
+    - name: Chad Kreutzer
+      image: https://avatars1.githubusercontent.com/u/13308917?v=3&s=400
+      github_url: https://github.com/ChadKreutzer
+    - name: Roberto Pando
+      image: https://avatars1.githubusercontent.com/u/20132181?v=3&s=400
+      github_url: https://github.com/RobPando
+    - name: Ryan
```

Comment:
I'm still seeing a smol Ryan avatar, but swapping him and Rob so Ryan is second last in the list seems to sort it out

Reply:
Working now post the `.erb` fix :tada: